src/main/java/at/hannibal2/skyhanni/utils/renderables/Renderable.kt

```diff
@@ -47,6 +60,9 @@ interface Renderable {
             }
         }
 
+        enum class HorizontalAlignment { Left, Center, Right }
+        enum class VerticalAlignment { Top, Center, Bottom }
```

Comment:
hmm, thank you for making me aware that I missed to update a branch of mine. Is fixed if you merge with #887 again.